db/pre-re/sc_config.conf

```diff
@@ -5156,6 +5156,7 @@ SC_GENTLETOUCH_CHANGE: {
 		Maxhp: true
 		Mdef: true
 		Aspd: true
+		Watk: true
```

Comment:
please keep these lists sorted to reduce confusion when auto generating them. (move `Watk` under `Maxhp`).